package.json

```diff
@@ -35,7 +35,7 @@
   },
   "dependencies": {
     "codemirror": "^5.1.0",
-    "devtools-launchpad": "0.0.23",
+    "devtools-launchpad": "0.0.24",
```

Comment:
We need some kind of yarn-lint that checks we've updated the `yarn.lock` file whenever the `package.json` updates.